_docker/drupal/drupal-filesystem/composer.json

```diff
@@ -81,7 +45,7 @@
         "drupal/console-core": "1.0.0-rc10",
         "drupal/console-en": "1.0.0-rc10",
         "drupal/drupal-extension": "^3.2",        
-        "drush/drush": "8.1.8",
+        "drush/drush": "^8.1.8",
```

Comment:
Apparently, they pulled the "skip-modules" feature in cim and cex in 8.1.11, because it never worked anyway. Unfortunately, they also included some fixes for cim and cex which may be helpful for us :( I'm happy to pin this to 8.1.10 for this, but we should look at being able to move toward 8.1.11 soon.